src/Tizen.NUI/src/public/WindowSystem/QuickPanelClient.cs

```diff
@@ -0,0 +1,394 @@
+﻿/*
+ * Copyright(c) 2017 Samsung Electronics Co., Ltd.
```

Comment:
Please update the year in the the files.

Reply:
I changed year 2017 -> 2020